src/aws-cpp-sdk-core/include/aws/core/http/URI.h

```diff
@@ -71,12 +71,14 @@ namespace Aws
             void SetScheme(Scheme value);
 
             /**
-            * Gets the domain portion of the uri
+            * Gets the authority portion of the URI. Differs from the authority definition in RFC 3986. user information
```

Comment:
nit: maybe add that its essentially host part of authority with encoding still present